dune/xt/la/container/common.hh

```diff
@@ -762,49 +837,63 @@ public:
   } // CommonSparseMatrix(...)
 
   template <int ROWS, int COLS>
-  explicit operator Dune::FieldMatrix<ScalarType, ROWS, COLS>() const
+  explicit operator std::unique_ptr<Dune::FieldMatrix<ScalarType, ROWS, COLS>>() const
```

Comment:
Ok, makes sense! Nevertheless, I would add the operator without the ptr, to be usable for smaller matrices...